src/index.html

```diff
@@ -9,81 +9,48 @@
   <body>
     <h1>List of employees</h1>
     <ul>
-      <li
-        data-position="Accountant"
-        data-salary="$162,700"
-        data-age="33"
-      >
+      <li data-position="Accountant" data-salary="$162,700"
+       data-age="33">
         Airi Satou
       </li>
-      <li
-        data-position="Javascript Developer"
-        data-salary="$183,000"
-        data-age="29"
-      >
+      <li data-position="Javascript Developer" data-salary="$183,000"
+       data-age="29">
         Michael Bruce
       </li>
-      <li
-        data-position="Marketing Designer"
-        data-salary="$198,000"
-        data-age="66"
-      >
+      <li data-position="Marketing Designer" data-salary="$198,000"
+       data-age="66">
         Michael Silva
       </li>
-      <li
-        data-position="Technical Author"
-        data-salary="$145,000"
-        data-age="27"
-      >
+      <li data-position="Technical Author" data-salary="$145,000"
+       data-age="27">
         Prescott Bartlett
       </li>
-      <li
-        data-position="Developer"
-        data-salary="$98,540"
-        data-age="61"
-      >
+      <li data-position="Developer" data-salary="$98,540"
+       data-age="61">
         Thor Walton
       </li>
-      <li
-        data-position="Integration Specialist"
-        data-salary="$327,900"
-        data-age="55"
-      >
+      <li data-position="Integration Specialist" data-salary="$327,900"
+       data-age="55">
         Rhona Davidson
       </li>
-      <li
-        data-position="Data Coordinator"
-        data-salary="$138,575"
-        data-age="64"
-      >
+      <li data-position="Data Coordinator" data-salary="$138,575"
+       data-age="64">
         Serge Baldwin
       </li>
-      <li
```

Comment:
![image](https://user-images.githubusercontent.com/70009205/208722233-b3766321-d57a-4728-98b4-7d0f95537965.png)

Reply:
$ git push origin develop
remote: Permission to mate-academy/js_task_parse_list_DOM.git denied to aliliolek.
fatal: unable to access 'https://github.com/mate-academy/js_task_parse_list_DOM.git/': The requested URL returned error: 403